### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#465] [Esami & Esoneri M-Z]
### Traccia

<img width="933" alt="image" src="https://github.com/user-attachments/assets/571150df-c23a-4eff-be0c-6cdae4ea50ef">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/e32ffa1b-ea69-479f-8b58-c65b9df29664)
